### PR TITLE
Braze Currents documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Documentation for the Braze Currents source. Will do a final grammar cleanup after this review, in case there is any change in the content.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: stream-sources/braze-currents.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+# Braze
+
+[**Braze**](https://www.braze.com/) is an industry-leading customer engagement platform. It provides a cross-channel, data-driven platform to help you better understand your customers' in-app behavior, use the insights to improve your users' app experience, and increase engagement with them.
+
+You can send your Braze events to RudderStack by leveraging [**Braze Currents**](https://www.braze.com/docs/user_guide/data_and_analytics/braze_currents/).
+
+<div class="infoBlock">
+
+Braze Currents is available in select Braze packages and can be configured only with the Braze team's assistance. For support, you can contact the Braze Customer Success team <a href="https://www.braze.com/docs/support_contact/">here</a>.
+</div>
```
Comment (saikumarrs) [inline stream-sources/braze-currents.mdx:15]: This is already covered as part of the next section. Let's remove it?

File: stream-sources/braze-currents.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+| Email Unsubscribed | When the user clicks on the unsubscribe link within an email. | 
+| Subscription Group State Changed | When the user's subscription group status changes to `Subscribed` or `Unsubscribed`. |
+| Push Notification Sent | When a push notification is successfully sent to the user. | 
+| Push Notification Tapped | When a user taps on a push notification. | 
+| iOS Foreground Push Opened | When the user receives an iOS push notificaiton while the app is open. | 
+| In-App Message Viewed | When the user views an in-app message. | 
+| News Feed Viewed | When the user views the Braze News Feed. |
+| News Feed Card Viewed | When the user views a Card within the Braze News Feed. | 
+| News Feed Card Clicked | When the user taps or clicks on Braze News Feed Card. | 
+| Webhook Sent | When a webhook message is sent. |
```
Comment (saikumarrs) [inline stream-sources/braze-currents.mdx:70]: @arnab-p I believe we do not support the full list mentioned here. Can you please share the subset?
Accordingly, the next section needs to be cleaned.

File: stream-sources/braze-currents.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply stream-sources/braze-currents.mdx]: Sure - this is done.

File: stream-sources/braze-currents.mdx (hunk 2 above)
Comment (saikumarrs) [reply stream-sources/braze-currents.mdx:65]: @arnab-p I was referring to this comment in the code, https://github.com/rudderlabs/rudder-transformer/blob/5c85e75ea0293a49823f471f1c0513f763103a5d/v0/sources/braze/transform.js#L32

Comment (ameypv-rudder) [reply stream-sources/braze-currents.mdx:65]: Based on the outcome of the discussion, I can update the table accordingly.

Comment (arnab-p) [reply stream-sources/braze-currents.mdx:65]: We don't have any specific mapping for the event names. Thus, it's not a harm to list down all the events that Braze supports. The comment is somewhat misleading IMO.

**Review by arnab-p — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

File: stream-sources/braze-currents.mdx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+2. Assign a name to your source and click on **Next**.
+3. Your Braze source is now configured. Note the source **Write key** as it will be required in the later steps.
+
+<img src="../assets/event-stream-sources/braze-2.png" alt="Braze source write key" />
+
+4. To complete the setup, contact Braze's support team to activate Currents in your Braze account.
+
+<div class="infoBlock">
+
+Braze Currents is available in select Braze packages and can be configured only with the Braze team's assistance. For support, you can contact the Braze Customer Success team <a href="https://www.braze.com/docs/support_contact/">here</a>.
```
Comment (ishachopra0401) [inline stream-sources/braze-currents.mdx:28]:
```suggestion
Braze Currents is available in select Braze packages and can be configured only with the Braze team's assistance. For support, you can contact the <a href="https://www.braze.com/docs/support_contact/">Braze Customer Success team</a>.
```

>Braze Currents is available in select Braze packages

could not quite understand this.. like some enterprise plan package? not clear what sort of packages contain Braze Currents

**Review by ishachopra0401 — CHANGES_REQUESTED** [review]

File: stream-sources/braze-currents.mdx (hunk 3 above)
Comment (ameypv-rudder) [reply stream-sources/braze-currents.mdx:28]: Refer to [this](https://www.braze.com/docs/user_guide/data_and_analytics/braze_currents/#:~:text=A%20Currents%20connector%20is%20already%20included%20in%20many%20of%20our%20pro%2D%20and%20enterprise%2Dlevel%20packages) link for more context.

Comment (ishachopra0401) [reply stream-sources/braze-currents.mdx:28]: got it. In that case, maybe we can mention 'Braze Currents is available in many pro and enterprise-level Braze packages and....'

**Pull request merged**